roundManager.js

```diff
@@ -0,0 +1,100 @@
+'use strict'
+
+let io
+
+const NUM_ROUNDS = 5
+let round = 0
+let allClients
+let clientsLeft
+let activePlayerID
+let activePlayer
+let state
+
+function containsObject(obj, list) {
+  var i
+  for (i = 0; i < list.length; i++) {
+    if (list[i] === obj) {
+      return true
+    }
+  }
+
+  return false
+}
+
+module.exports = {
+  get activePlayerID() {
+    return activePlayerID
+  },
+  get state() {
+    return state
+  },
+  start(ioIn, room, users) {
+    io = ioIn
+    io.in(room).clients((err, clients) => {
+      if (err) {
+        throw err
+      }
+      allClients = clients
+      this.nextRound(users)
+    })
+  },
+
+  nextRound(users) {
+    round += 1
+    if (round <= NUM_ROUNDS) {
+      this.startChat(users)
+    } else {
+      this.endGame()
+    }
+  },
+
+  startChat(users) {
+    clientsLeft = allClients.slice(0)
+    this.continueChat(users)
+  },
+
+  continueChat(users) {
+    const randomPlayer = clientsLeft[Math.floor(Math.random() * clientsLeft.length)]
+    activePlayerID = randomPlayer
+    if (activePlayerID !== undefined) {
```

Comment:
Why not move the clientsLeft check (currently at the end of this function) to the start and just switch to voting if nobody is left?